src/components/Data/Documents/ListActions.vue

```diff
@@ -28,16 +26,46 @@
         Delete
       </button>
     </div>
+    <div v-if="viewType === 'map'" class="col s2">
```

Comment:
Wouldn't it be better to encapsulate this in a single `<div>` with just one `v-if`?